doc/api/buffer.md

```diff
@@ -788,6 +788,29 @@ const buf = Buffer.from([0x62, 0x75, 0x66, 0x66, 0x65, 0x72]);
 
 A `TypeError` will be thrown if `array` is not an `Array`.
 
+`Buffer.from(array)` will also use the internal `Buffer` pool if `array.length`
+is less than or equal to half [`Buffer.poolSize`] just like
+[`Buffer.allocUnsafe()`] does.
```

Comment:
This makes it sound like using the internal pool is guaranteed in that situation, but it's not, is it?

Reply:
s/will/can/? Also I don't think that we should specifically document those length conditions — those are optimization heuristics and could change.

Reply:
Maybe `may` since that's what is used in the other text about this in the doc? Just keep it all consistent for find/replace purposes etc.